src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $hover-transition;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
   }
+
+  &__card:hover {
+    transform: scale(120%);
+    color: #34568b;
+  }
```

Comment:
You can nest hover inside card style. 
```
&__card {
  &:hover {
    transform: scale(120%);
    color: #34568b;
  }
}
```
Also better use the coefficient for scale(), not presents I guess.

[checklist p3](https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md)